operator/pkg/apis/cassandra/v1beta1/webhook.go

```diff
@@ -35,24 +45,24 @@ func ValidateSingleDatacenter(dc CassandraDatacenter) error {
 func ValidateDatacenterFieldChanges(oldDc CassandraDatacenter, newDc CassandraDatacenter) error {
 
 	if oldDc.Spec.ClusterName != newDc.Spec.ClusterName {
-		return errors.New("CassandraDatacenter attempted to change ClusterName")
+		return attemptedTo("change ClusterName")
```

Comment:
getting nitpicky, do you think this should match the way users see the yaml? so `clusterName`, if so. also all the ones below here, if you agree.